tetrateci/test_1.8.sh

```diff
@@ -8,40 +8,47 @@ if [[ ${CLUSTER} == "gke" ]]; then
-go test -count=1 -tags=integ ./tests/integration/pilot/analysis/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/pilot/revisions/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/telemetry/requestclassification/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/telemetry/outboundtrafficpolicy/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/ca_custom_root/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/ecc_signature_algorithm/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/chiron/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/filebased_tls_origination/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/mtls_first_party_jwt/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/mtlsk8sca/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/sds_egress/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/sds_tls_origination/... -istio.test.skipVM true -p 1 -test.v
-go test -count=1 -tags=integ ./tests/integration/security/webhook/... -istio.test.skipVM true -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/helm/...  -p 1 -test.v
+
+go test -count=1 -tags=integ ./tests/integration/operator/...   -p 1  -test.v
+
+go test -count=1 -tags=integ ./tests/integration/pilot/analysis/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/pilot/revisions/...  -p 1 -test.v
+
+go test -count=1 -tags=integ ./tests/integration/telemetry/requestclassification/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/telemetry/outboundtrafficpolicy/...  -p 1 -test.v
+
+go test -count=1 -tags=integ ./tests/integration/security/ca_custom_root/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/ecc_signature_algorithm/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/chiron/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/filebased_tls_origination/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/mtls_first_party_jwt/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/mtlsk8sca/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/sds_egress/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/sds_tls_origination/...  -p 1 -test.v
+go test -count=1 -tags=integ ./tests/integration/security/webhook/...  -p 1 -test.v
 
 if [[ ${CLUSTER} == "eks" ]]; then
```

Comment:
The series of if and else is less readable. Let's create add the cluster specific patches to the top of the script. Then the common tests. Then cluster specific tests. Lets have 3 if blocks to run cluster specific tests. That way it will more readable. WDYT?

Reply:
Right, also clubbing the tests from specific modules together.